### DISCUSSION TITLE
Network configuration

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#541] [Q&A] [answered]
I have a couple of questions about network configuration:

1) We have an internal non-routable network with ISC DHCP server. The server doesn't provide a default gateway for the clients.  When LTSP client boots it shows an error:

```
RTNETLINK answers: Network is unreachable
LTSP command failed: ip -o route get 192.168.67.1
Could not parse output of: ip -o route get 192.168.67.1
```

I don't know where 192.168.67.1 comes from (I guess it's some hardcoded LTSP thing?), everything else works fine except this error. And it dissapears if I configure DHCP to provide a gateway, even a false one.
I was just wondering what this means and if you need a gateway for LTSP to work properly.

2) How can I enable dynamic DNS updates? Our DHCP server is configured to update DNS records but it seems LTSP clients don't send their hostnames when they make DHCP request. I tried to add "send host-name" to dhclient.conf but it didn't work.

**Comment by alkisg** [comment, marked as answer]
1. It's a warning, not an error, you may safely ignore it. With that command, LTSP is trying to detect the "internal" interface of LTSP servers with 2 NICs.
2. Debian doesn't use dhclient for netbooting, it uses `/usr/lib/klibc/bin/ipconfig`. This program doesn't support DNS updates nor DHCP lease updates. It wouldn't make a lot of sense to try to change it, since netbooted clients can't handle lease expiration, and they do not have a DNS name before contacting DHCP. You may do the opposite, you may configure DNS names and static DHCP leases in your servers.
